ge/org.osate.ge.ba/src/org/osate/ge/ba/util/BehaviorAnnexXtextUtil.java

```diff
@@ -48,14 +48,19 @@ public static String getText(final IXtextDocument xtextDocument, final XtextReso
 	/**
 	 * Find offset of character in string that is not commented out
 	 */
-	public static int findUncommentedChar(final String str, final char delim) {
+	public static int findUncommentedTerminationChar(final String str, final char delim) {
```

Comment:
Add javadoc for this function which contains a description of the function, parameters, resulting return value/exception.